src/routers/comment-routers.js

```diff
@@ -0,0 +1,38 @@
+const router = require("express").Router();
+const Comments = require("../database/models/comment-models");
```

Comment:
Model name should be singular, e.g `Comment`

Reply:
Changed to singular